LPS-186788 Remove multiple configuration from LiferayOauthDataProvider

julschong opened the pull request:
resubmitted from #4858

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-38.liferay.com/job/test-portal-source-format/4713/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/4713/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90">9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/julschong/liferay-portal/tree/LPS-186788-multi-config-LiferayOauthDataProvider-6-23-1">LPS-186788-multi-config-LiferayOauthDataProvider-6-23-1</a><br/>Branch GIT ID: <a href="https://github.com/julschong/liferay-portal/commit/ca4d7cbf69baa3167369333876ebe5a5bc8fcb8f">ca4d7cbf69baa3167369333876ebe5a5bc8fcb8f</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-38.liferay.com/job/test-portal-source-format/4713/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/4713/jenkins-report.html">here</a>.</h5></details></html>

julschong commented:
ci:test:oauth2

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-source-format/4713/">test-portal-source-format#4713</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-source-format/builds/4713/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-core-infra/liferay-portal/pull/4895">liferay-core-infra#4895</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2364132011">[master] ci:test:sf - julschong &gt; liferay-core-infra - PR#4895 - 2023-06-28[15:14:32]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/16990/">publish-testray-report#16990</a><br/></div></div>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>oauth2</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8938/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/4328/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:oauth2 - 51 out of 57 jobs passed in 1 hour 35 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4657">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1277401_8938</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90">9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/44ca914e99f5e423589bf43ea9a885a2d14eba83">44ca914e99f5e423589bf43ea9a885a2d14eba83</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2364217585">EE Development Acceptance (master) - 4656 - 2023-06-28[12:41:38]</a></p><details><summary><strong>ci:test:oauth2 - 51 out of 57 jobs PASSED</strong></summary><div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8938/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/623016/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611589/">functional-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205644/">functional-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179927/">functional-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707170/">functional-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610199/">functional-tomcat90-mysql57-jdk8/1/3</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555638/">functional-tomcat90-mysql57-jdk8/1/4</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680447/">functional-tomcat90-mysql57-jdk8/1/5</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205645/">functional-tomcat90-mysql57-jdk8/1/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611586/">functional-tomcat90-mysql57-jdk8/1/7</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179928/">functional-tomcat90-mysql57-jdk8/1/8</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456518/">functional-tomcat90-mysql57-jdk8/1/9</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680448/">functional-tomcat90-mysql57-jdk8/1/10</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555639/">functional-tomcat90-mysql57-jdk8/1/11</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610200/">functional-tomcat90-mysql57-jdk8/1/12</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707171/">functional-tomcat90-mysql57-jdk8/1/13</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179929/">functional-tomcat90-mysql57-jdk8/1/14</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611587/">functional-tomcat90-mysql57-jdk8/1/15</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680449/">functional-tomcat90-mysql57-jdk8/1/16</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179923/">functional-tomcat90-mysql57-jdk8/1/17</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179924/">functional-tomcat90-mysql57-jdk8/1/18</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179925/">functional-tomcat90-mysql57-jdk8/1/19</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179926/">functional-tomcat90-mysql57-jdk8/1/20</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680446/">functional-tomcat90-mysql57-jdk8/1/21</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555637/">functional-tomcat90-mysql57-jdk8/1/22</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707168/">functional-tomcat90-mysql57-jdk8/1/23</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456517/">functional-tomcat90-mysql57-jdk8/1/24</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/584310/">functional-tomcat90-mysql57-jdk8/2/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240362/">functional-tomcat90-mysql57-jdk8/2/1</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544189/">functional-tomcat90-mysql57-jdk8/2/2</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205648/">functional-tomcat90-mysql57-jdk8/2/3</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/238110/">functional-tomcat90-mysql57-jdk8/2/4</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/592458/">functional-tomcat90-mysql57-jdk8/2/5</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461137/">functional-tomcat90-mysql57-jdk8/2/6</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/581896/">functional-tomcat90-mysql57-jdk8/2/7</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/553357/">functional-tomcat90-mysql57-jdk8/2/8</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707167/">functional-tomcat90-mysql57-jdk8/2/9</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707169/">functional-tomcat90-mysql57-jdk8/2/10</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456520/">functional-tomcat90-mysql57-jdk8/2/11</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/584309/">functional-tomcat90-mysql57-jdk8/2/12</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205647/">functional-tomcat90-mysql57-jdk8/2/13</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/641595/">functional-tomcat90-mysql57-jdk8/2/14</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456519/">js-unit-jdk8/0/0</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190/"><strike>modules-integration-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/641596/"><strike>modules-integration-mysql57-jdk8/0/1</strike></a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610201/"><strike>modules-semantic-versioning-jdk8/0/0</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555640/"><strike>modules-semantic-versioning-jdk8/0/1</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179931/">modules-semantic-versioning-jdk8/0/2</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456521/">modules-semantic-versioning-jdk8/0/3</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610202/">modules-semantic-versioning-jdk8/0/4</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555641/">modules-semantic-versioning-jdk8/0/5</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680450/">modules-semantic-versioning-jdk8/0/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611588/"><strike>modules-semantic-versioning-jdk8/0/7</strike></a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/179930/">modules-semantic-versioning-jdk8/0/8</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707172/">modules-unit-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205646/">semantic-versioning-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>51 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190//consoleText">modules-integration-mysql57-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190/testReport/com.liferay.oauth2.provider.client.test/HttpMethodApplicationClientTest">com.liferay.oauth2.provider.client.test.HttpMethodApplicationClientTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630289/history">Failed 1 of last 23</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190//testReport/com.liferay.oauth2.provider.client.test/HttpMethodApplicationClientTest/test">test</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/641596//consoleText">modules-integration-mysql57-jdk8/0/1</a><ol><li><details><summary><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/641596/testReport/com.liferay.oauth2.provider.token.endpoint.test/IssueJWTAccessTokenTest">com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630382/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/641596//testReport/com.liferay.oauth2.provider.token.endpoint.test/IssueJWTAccessTokenTest/testIssuedJWTAccessToken">testIssuedJWTAccessToken</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 44ca914e99f5e423589bf43ea9a885a2d14eba83:</strong></summary><ol><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190//consoleText">modules-integration-mysql57-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190/testReport/com.liferay.oauth.client.test/LocalOAuthClientTest">com.liferay.oauth.client.test.LocalOAuthClientTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630370/history">Failed 23 of last 23</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190//testReport/com.liferay.oauth.client.test/LocalOAuthClientTest/test">test</a></li></ol></div></details></li><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544190//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610201//consoleText">modules-semantic-versioning-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610201/testReport/com.liferay.semantic.versioning/SemanticVersioningTest">com.liferay.semantic.versioning.SemanticVersioningTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610201//testReport/com.liferay.semantic.versioning/SemanticVersioningTest/testSemanticVersioning__apps_client_extension_client_extension_api_">testSemanticVersioning[/apps/client-extension/client-extension-api]</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610201//testReport/com.liferay.semantic.versioning/SemanticVersioningTest/testSemanticVersioning__apps_client_extension_client_extension_type_api_">testSemanticVersioning[/apps/client-extension/client-extension-type-api]</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555640//consoleText">modules-semantic-versioning-jdk8/0/1</a><ol><li><details><summary><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555640/testReport/com.liferay.semantic.versioning/SemanticVersioningTest">com.liferay.semantic.versioning.SemanticVersioningTest</a></summary><div><ol><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555640//testReport/com.liferay.semantic.versioning/SemanticVersioningTest/testSemanticVersioning__apps_frontend_taglib_frontend_taglib_clay_">testSemanticVersioning[/apps/frontend-taglib/frontend-taglib-clay]</a></li></ol></div></details></li></ol></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/8938/">test-portal-acceptance-pullrequest(master)#8938</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/8938/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>oauth2<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-core-infra/liferay-portal/pull/4895">liferay-core-infra#4895</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2364286848">[master] ci:test:oauth2 - julschong &gt; liferay-core-infra - PR#4895 - 2023-06-28[15:33:12]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/19829/">publish-testray-report#19829</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 32 out of 32 jobs passed</h3><h3>:x: ci:test:relevant - 72 out of 78 jobs passed in 2 hours 22 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/4657">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1347401_4328</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90">9896b7ac7511fbd660ec2bb8afca63ebe5f7ef90</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/44ca914e99f5e423589bf43ea9a885a2d14eba83">44ca914e99f5e423589bf43ea9a885a2d14eba83</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2364217585">EE Development Acceptance (master) - 4656 - 2023-06-28[12:41:38]</a></p><details><summary><strong>ci:test:stable - 32 out of 32 jobs PASSED</strong></summary><details><summary><strong>32 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 72 out of 78 jobs PASSED</strong></summary><div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/4328/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680510/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/185892/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544234/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/218552/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240636/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/221630/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223489/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/214846/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/581947/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/229410/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456588/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/227023/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/553400/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/0</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611647/">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224205/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611648/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/218554/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/581951/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223493/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/231080/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/244947/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680514/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/227022/">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223491/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/231079/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240385/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224204/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/221631/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/222204/">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/222203/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240638/">functional-upgrade-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/227930/">functional-upgrade-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/231078/">functional-upgrade-tomcat90-mysql57-jdk8/0/9</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/214845/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228278/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240637/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680512/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/611644/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223488/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/5</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707233/">functional-upgrade-tomcat90-postgresql144-jdk8/0/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461193/">functional-upgrade-tomcat90-postgresql144-jdk8/0/1</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/244946/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-mysql57-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461190/"><strike>lpkg-controller-mysql57-jdk8/0/0</strike></a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610249/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555688/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/223486/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/218126/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/231077/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461191/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240384/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/610250/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555690/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456589/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461188/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461189/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555698/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680517/"><strike>modules-integration-mysql57-jdk8/0/1</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544235/"><strike>modules-integration-mysql57-jdk8/0/2</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456591/"><strike>modules-integration-mysql57-jdk8/0/3</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461194/">modules-integration-mysql57-jdk8/0/4</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/584363/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/707234/">modules-integration-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/553401/">modules-integration-mysql57-jdk8_stable/0/2</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456590/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/185891/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555695/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/244944/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/555689/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/227929/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/592517/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/205705/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/240640/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/581950/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/244945/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/222205/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/623059/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/228277/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>72 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/461190//consoleText">lpkg-controller-mysql57-jdk8/0/0</a><pre>move-artifact:
[stopwatch] [run.batch.test.action: 1:18.772 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:365: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1974: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:2782: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/modules/build-app.xml:208: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/modules/build-app.xml:210: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/modules/build-app.xml:390: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/modules/build-app-module.xml:438: Warning: Could not find file /opt/dev/projects/github/liferay-portal/bundles/osgi/modules/com.liferay.bookmarks.api.jar to copy.
      [get] Getting: \$\{dependencies.url\}/archive/test-1-33/test-portal-acceptance-pullrequest-downstream(master)/461190consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20230628162651828.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20230628162651828.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null897792927.properties</pre></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680517//consoleText">modules-integration-mysql57-jdk8/0/1</a><ol><li><details><summary><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680517/testReport/com.liferay.oauth2.provider.client.test/AnnotationsAndHttpApplicationClientTest">com.liferay.oauth2.provider.client.test.AnnotationsAndHttpApplicationClientTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630390/history">Failed 2 of last 23</a></summary><div><ol><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/680517//testReport/com.liferay.oauth2.provider.client.test/AnnotationsAndHttpApplicationClientTest/test">test</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544235//consoleText">modules-integration-mysql57-jdk8/0/2</a><ol><li><details><summary><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544235/testReport/com.liferay.oauth2.provider.client.test/SAPClientTest">com.liferay.oauth2.provider.client.test.SAPClientTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630313/history">Failed 2 of last 24</a></summary><div><ol><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/544235//testReport/com.liferay.oauth2.provider.client.test/SAPClientTest/test">test</a></li></ol></div></details></li></ol></div></li><li><div><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456591//consoleText">modules-integration-mysql57-jdk8/0/3</a><ol><li><details><summary><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456591/testReport/com.liferay.oauth2.provider.token.endpoint.test/IssueJWTAccessTokenTest">com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2363630382/history">Failed 0 of last 24</a></summary><div><ol><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/456591//testReport/com.liferay.oauth2.provider.token.endpoint.test/IssueJWTAccessTokenTest/testIssuedJWTAccessToken">testIssuedJWTAccessToken</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 44ca914e99f5e423589bf43ea9a885a2d14eba83:</strong></summary><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/553400//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a><pre>[beanshell] "Attach Listener" #180 daemon prio=9 os_prio=0 tid=0x00007f0fb0001000 nid=0x1943dd waiting on condition [0x0000000000000000]
[beanshell]    java.lang.Thread.State: RUNNABLE
[beanshell] 
[beanshell]    Locked ownable synchronizers:
[beanshell]   - None
[beanshell] 
[beanshell] "liferay/scheduled_user_ldap_import-2" #173 daemon prio=5 os_prio=0 tid=0x00007f0ee8001000 nid=0x1942ae waiting on condition [0x00007f0f1a824000]
[beanshell] 
   [delete] Deleting: /tmp/Bootstrap__1655004.log
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/jps_output
     [echo] stop.app.server.0.start.timestamp: 06-28-2023 17:08:40:317 PDT
[stopwatch] [stop.app.server.0: 8.492 sec]
[stopwatch] [test.execution.duration: 34:30.881 sec]
[stopwatch] [run.batch.test.action: 35:01.681 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:365: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1648: /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1652: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-tomcat.xml:40: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-tomcat.xml:52: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:15802: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:16175: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:6910: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:6987: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:8265: No server is available at http://localhost:8080/web/guest
      [get] Getting: \$\{dependencies.url\}/archive/test-1-35/test-portal-acceptance-pullrequest-downstream(master)/553400consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20230628170848822.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20230628170848822.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1035355160.properties</pre></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-34/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-34/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-34/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest(master)/4328/">test-portal-acceptance-pullrequest(master)#4328</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-34.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4328/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-core-infra/liferay-portal/pull/4895">liferay-core-infra#4895</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2364316479">[master] ci:test:relevant - julschong &gt; liferay-core-infra - PR#4895 - 2023-06-28[15:34:52]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-34.liferay.com/job/publish-testray-report/9955/">publish-testray-report#9955</a><br/></div></div>

Kvale1733 commented:
@julschong Regression found. This looks like it was present in the previous pr to btw but was missed.

com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest
```
exec] com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest > testIssuedJWTAccessToken FAILED
     [exec]     java.lang.AssertionError
     [exec]     	at org.junit.Assert.fail(Assert.java:87)
     [exec]     	at org.junit.Assert.assertTrue(Assert.java:42)
     [exec]     	at org.junit.Assert.assertTrue(Assert.java:53)
     [exec]     	at com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest._verifyJWTAccessToken(IssueJWTAccessTokenTest.java:67)
     [exec]     	at com.liferay.oauth2.provider.token.endpoint.test.IssueJWTAccessTokenTest.testIssuedJWTAccessToken(IssueJWTAccessTokenTest.java:44)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```
[Log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-34/test-portal-acceptance-pullrequest(master)/4328/modules-integration-mysql57-jdk8/0/3/jenkins-console.txt.gz?authuser=0)

julschong commented:
seems like it's kinda flaky. I start got it to show up to working following this https://liferay.atlassian.net/wiki/spaces/~85516578/pages/2369454549/Integration+Tests

but after I cleaned and deleted all the tables and restarted, it never showed again :|
